src/vcStringFormat.cpp

```diff
@@ -56,7 +57,7 @@ const char *vcStringFormat(const char *pFormatString, const char **ppStrings, si
     // Format Specifier
     if (pFormatString[i] == '{')
     {
-      if (len <= i + 1 || pFormatString[i + 1] < '0' || pFormatString[i + 1] > '9')
+      if (pFormatString[i + 1] == '\0' || pFormatString[i + 1] < '0' || pFormatString[i + 1] > '9')
```

Comment:
I'm fairly certain that `\0` lies outside of `0` and `9`, this seems like a redundant check? Is this for the verbosity?

Reply:
just didn't occur to me because I was only reworking the previous line to check the length without having the length